### PR TITLE
Add CHANGELOG.md to prepare for release.

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

**Comment by waywardmonkeys** [comment]
Actually, I think I want to change / improve the text here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+-->
+
+## Unreleased
+
+### Added
+
+- [#26](https://github.com/linebender/peniko/pull/26) - Add serde derives by [@ratmice](https://github.com/ratmice)
+
+## 0.1 (2024-02-15)
```
Comment (xStrom) [inline CHANGELOG.md:17]: Let's spell out the full version.

```suggestion
## 0.1.0 (2024-02-15)
```

**Comment by xStrom** [comment]
Is the plan to wait for some other PRs before release?

If not, then might as well finalize the changelog with 0.1.1, the date, and and an empty unreleased section.

**Comment by waywardmonkeys** [comment]
I don't see much point in waiting since there's not yet a consensus about them and we have a user who would like the serde changes.

My thought was the same: A separate PR that does everything for updating to 0.1.1 would be best.

**Review by xStrom — APPROVED** [review]
Thanks for getting a changelog going!

**Pull request merged**